bikespace_frontend/src/components/landing/content-blocks/content-blocks.module.scss

```diff
@@ -17,11 +17,14 @@
   margin-top: 2rem;
   margin-bottom: 2rem;
   max-width: $wrapper-full-width;
+  flex: 2 1 400px;
+  padding: 5%;
```

Comment:
If you want more padding between the content and screen edge, I would suggest adjusting `bikespace_frontend/src/components/landing/landing-layout/landing-layout.module.scss` (line 11).

There's also a knock-on effect that with the greater padding for this component, it causes the hero button text to break into two lines on mobile:

<img src="https://github.com/user-attachments/assets/25f78777-3cd9-4c5c-b596-165ba201b25d" width=400>

Reply:
Updated